log/log.go

```diff
@@ -39,6 +59,18 @@ func GetOSSigChannel() chan os.Signal {
 	return c
 }
 
+func RegexCompile(o Options) {
+
+	CNamespace, _ = regexp.Compile(o.Namespace)
+	CLogtype, _ = regexp.Compile("(?i)" + o.LogType)
+	COperation, _ = regexp.Compile("(?i)" + o.Operation)
+	CContainerName, _ = regexp.Compile(o.ContainerName)
+	CPodName, _ = regexp.Compile(o.PodName)
+	CSource, _ = regexp.Compile(o.Source)
+	CResource, _ = regexp.Compile(o.Resource)
```

Comment:
We should return error handle them, just in case there's a invalid regex provided we should return the error saying the same.